library_api/controllers/book_controller.go

```diff
@@ -0,0 +1,75 @@
+package controllers
```

Comment:
I would have renamed the file to just book, because we already know that it is controller (in controller package)